README.md

```diff
@@ -5,6 +5,17 @@ using Yocto Project tools and the meta-tegra BSP layer.
 
 ![Build status](https://builder.madison.systems/badges/tegrademo-dunfell.svg)
```

Comment:
This needs updating too - the correct URL is https://builder.madison.systems/badges/tegrademo-dunfell-32-5-0.svg

Reply:
I will update it right now

Reply:
Done